Documentation/DocuBlocks/Rest/Administration/get_admin_modules_flush.md

```diff
@@ -126,6 +126,30 @@ One of the possible log levels.
 @RESTBODYPARAM{agencycomm,string,optional,string}
 One of the possible log levels.
 
+@RESTBODYPARAM{aql,string,optional,string}
+One of the possible log levels.
+
+@RESTBODYPARAM{arangosearch,string,optional,string}
+One of the possible log levels.
+
+@RESTBODYPARAM{audit-authentication,string,optional,string}
+One of the possible log levels (_Enterprise Edition only_).
+
+@RESTBODYPARAM{audit-collection,string,optional,string}
+One of the possible log levels (_Enterprise Edition only_).
+
+@RESTBODYPARAM{audit-database,string,optional,string}
+One of the possible log levels (_Enterprise Edition only_).
+
+@RESTBODYPARAM{audit-document,string,optional,string}
```

Comment:
great catch. The log level is indeed called `audit-documentation` by mistake, then renamed here:
https://github.com/arangodb/arangodb/commit/428eed8a6a6fa9b34a4adc366b7c3dcaebf82d58#diff-36aa7a7b3c992b0cf77527acc3288e8d
and once more here:
https://github.com/arangodb/arangodb/commit/74edf938bcaec6534df8409432312a73704c99da#diff-36aa7a7b3c992b0cf77527acc3288e8d
but I can't find this in CHANGELOG!

```suggestion
@RESTBODYPARAM{audit-documentation,string,optional,string}
```